### PR TITLE
LPS-121454 Add '.../' to the title to indicate that is not the first level of a navigation menu

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/fragment/fragment-service/src/main/resources/com/liferay/fragment/internal/validator/dependencies/configuration-json-schema.json [hunk 1]
```diff
@@ -121,6 +121,7 @@
 						"collectionSelector",
 						"itemSelector",
 						"select",
+						"navigationMenuSelector",
```
Comment (ealonso) [inline modules/apps/fragment/fragment-service/src/main/resources/com/liferay/fragment/internal/validator/dependencies/configuration-json-schema.json:124]: @victorg1991 a b c d e f g ... s > n, 🤔

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/types/config.d.ts [hunk 2]
```diff
@@ -163,6 +163,7 @@ export interface Config {
 		};
 	};
 
+	siteNavigationMenuItemSelectorURL: string;
```
Comment (ealonso) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/types/config.d.ts:166]: @victorg1991 same here

File: modules/apps/fragment/fragment-service/src/main/resources/com/liferay/fragment/internal/validator/dependencies/configuration-json-schema.json (hunk 1 above)
Comment (victorg1991) [reply modules/apps/fragment/fragment-service/src/main/resources/com/liferay/fragment/internal/validator/dependencies/configuration-json-schema.json:124]: 🤪

File: modules/apps/site-navigation/site-navigation-item-selector-web/src/main/resources/META-INF/resources/view_site_navigation_menu_contextual_menus.jsp [hunk 3]
```diff
@@ -67,9 +68,18 @@ SiteNavigationMenuContextualMenusItemSelectorViewDisplayContext siteNavigationMe
...
 </clay:container-fluid>
 
-<aui:script>
-	Liferay.Util.selectEntityHandler(
-		'#<portlet:namespace />contextualMenuSelector',
-		'<%= HtmlUtil.escapeJS(siteNavigationMenuContextualMenusItemSelectorViewDisplayContext.getItemSelectedEventName()) %>'
-	);
-</aui:script>
\ No newline at end of file
+<liferay-frontend:component
```
Comment (ealonso) [inline modules/apps/site-navigation/site-navigation-item-selector-web/src/main/resources/META-INF/resources/view_site_navigation_menu_contextual_menus.jsp:71]: @victorg1991 Moves to display context

File: modules/apps/site-navigation/site-navigation-item-selector-web/src/main/resources/META-INF/resources/select_site_navigation_menu_level.jsp [hunk 4]
```diff
@@ -67,9 +67,18 @@ SelectSiteNavigationMenuDisplayContext selectSiteNavigationMenuDisplayContext =
...
 </div>
 
-<aui:script>
-	Liferay.Util.selectEntityHandler(
-		'#<portlet:namespace />siteNavigationMenuLevelSelector',
-		'<%= HtmlUtil.escapeJS(selectSiteNavigationMenuDisplayContext.getItemSelectedEventName()) %>'
-	);
-</aui:script>
\ No newline at end of file
+<liferay-frontend:component
```
Comment (ealonso) [inline modules/apps/site-navigation/site-navigation-item-selector-web/src/main/resources/META-INF/resources/select_site_navigation_menu_level.jsp:71]: @victorg1991 Moves to display context

File: modules/apps/site-navigation/site-navigation-item-selector-web/src/main/java/com/liferay/site/navigation/item/selector/web/internal/display/context/SelectSiteNavigationMenuDisplayContext.java [hunk 5]
```diff
@@ -86,6 +86,15 @@ public SelectSiteNavigationMenuDisplayContext(
 		return breadcrumbEntries;
 	}
 
+	public String getCurrentLevelTitle() throws Exception {
+		List<BreadcrumbEntry> breadcrumbEntries = getBreadcrumbEntries();
```
Comment (ealonso) [inline modules/apps/site-navigation/site-navigation-item-selector-web/src/main/java/com/liferay/site/navigation/item/selector/web/internal/display/context/SelectSiteNavigationMenuDisplayContext.java:96]: @victorg1991 Is not there any other way of doing that?

**Pull request closed without merging**